docker/scripts/get_image_digest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+VERSION=$1
+IMAGE=""
+
+if [ -z "${VERSION}" ]; then
+    echo "no version provided (i.e, v2.2.0), using latest tag"
+    VERSION="latest"
+fi
+
+for package in backend frontend; do
+  PACKAGE=mempool/"$package"
+  IMAGE="$PACKAGE":"$VERSION"
+  HASH=`docker pull $IMAGE > /dev/null && docker inspect $IMAGE | sed -n '/RepoDigests/{n;p;}' | grep -o '[0-9a-f]\{64\}'`
```

Comment:
```suggestion
  HASH=`docker pull $IMAGE > /dev/null && docker inspect $IMAGE | jq -r '.[].RepoDigests[0]' | cut -d @ -f2`
```

Reply:
this was intentional to avoid the dependency on `jq`